src/main/webapp/view/lecture-view.css

```diff
@@ -87,6 +87,11 @@ discussion-comment[highlighted] > [slot='header'] {
 
 /* Transcript */
 
+.scroll-banner {
+  background-color: #4c8bf5;
+  visibility: hidden;
```

Comment:
can you add `cursor: pointer;` so it feels like something that can be clicked?

Reply:
done, thanks!